wagtail_wordpress_import/test/tests/test_transform_styles_filter.py

```diff
@@ -0,0 +1,131 @@
+        transform_style_bold(soup, soup.find("span"))
+        bold = soup.find("b")
+        span = soup.find("span")
+
+        self.assertTrue(bold)
+        self.assertFalse(span)
+
+    def test_transform_style_italic(self):
+        input = '<span style="font-style: italic;">Text content</span>'
+        soup = normalize_style_attrs(BeautifulSoup(input, "html.parser"))
+        transform_style_italic(soup, soup.find("span"))
+        italic = soup.find("i")
+        span = soup.find("span")
+
+        self.assertTrue(italic)
+        self.assertFalse(span)
+
+    def test_transform_style_center(self):
+        input = '<p style="text-align: center;">Text content</p>'
+        soup = normalize_style_attrs(BeautifulSoup(input, "html.parser"))
+        transform_style_center(soup, soup.find("p"))
+        paragraph = soup.find("p")
+
+        self.assertTrue(paragraph)
+        self.assertEqual(paragraph.attrs["class"], "align-center")
+
+    def test_transform_style_left(self):
+        input = '<p style="text-align: left;">Text content</p>'
+        soup = normalize_style_attrs(BeautifulSoup(input, "html.parser"))
+        transform_style_left(soup, soup.find("p"))
+        paragraph = soup.find("p")
+
+        self.assertTrue(paragraph)
+        self.assertEqual(paragraph.attrs["class"], "align-left")
+
+    def test_transform_style_right(self):
+        input = '<p style="text-align: right;">Text content</p>'
+        soup = normalize_style_attrs(BeautifulSoup(input, "html.parser"))
+        transform_style_right(soup, soup.find("p"))
+        paragraph = soup.find("p")
+
+        self.assertTrue(paragraph)
+        self.assertEqual(paragraph.attrs["class"], "align-right")
+
+    def test_transform_float_left(self):
+        input = '<p style="float: left;">Text content</p>'
+        soup = normalize_style_attrs(BeautifulSoup(input, "html.parser"))
+        transform_float_left(soup, soup.find("p"))
+        paragraph = soup.find("p")
+
+        self.assertTrue(paragraph)
+        self.assertEqual(paragraph.attrs["class"], "float-left")
+
+    def test_transform_float_right(self):
+        input = '<p style="float: right;">Text content</p>'
+        soup = normalize_style_attrs(BeautifulSoup(input, "html.parser"))
+        transform_float_right(soup, soup.find("p"))
+        paragraph = soup.find("p")
+
+        self.assertTrue(paragraph)
+        self.assertEqual(paragraph.attrs["class"], "float-right")
+
+    def transform_html_tag_strong(self):
+        input = "<strong>Text content</strong>"
+        soup = BeautifulSoup(input, "html.parser")
+        transform_html_tag_strong(soup, soup.find("strong"))
+        strong = soup.find("b")
+
+        self.assertTrue(strong)
+
+    def transform_html_tag_em(self):
+        input = "<em>Text content</em>"
+        soup = BeautifulSoup(input, "html.parser")
+        transform_html_tag_em(soup, soup.find("em"))
+        italic = soup.find("i")
+
+        self.assertTrue(italic)
```

Comment:
**Suggestion**: Can we add a test that specifies a configuration, (e.g. including em but not strong) and then runs on an input, and asserts that the configuration is obeyed, e.g. that `filter_transform_inline_styles` acts on the em elements but not the strong ones? Then we're testing the ability to specify the configuration.

Reply:
I have implemented this test.